packages/spruce-types/src/gql/types/SprucebotAvatar.gql

```diff
@@ -0,0 +1,32 @@
+"Sprucebot's current state of mind"
+enum SprucebotAvatarStateOfMind {
+	"For when nothing in particular is happening"
```

Comment:
Let's make sure the voice of the docs here is consistent for all options (i.e. all should start with "For use when..." to be more prescriptive or "Nothing is happening right now" if more descriptive)